server/registry/server.go

```diff
@@ -29,12 +32,13 @@ import (
 
 // Config configures the registry server.
 type Config struct {
-	Database  string
-	DBConfig  string
-	LogLevel  string
-	LogFormat string
-	Notify    bool
-	ProjectID string
+	Database         string
+	DBConfig         string
+	LogLevel         string
+	LogFormat        string
+	Notify           bool
+	ProjectID        string
+	APIClientOptions []option.ClientOption
```

Comment:
What does this look like in YAML? There's a default config file in the `config` directory, could we modify that with something that illustrates this? (it could be commented out)

Reply:
At this point, this wouldn't be exposing this as it's only used by the tests to set up a local connection to the mock server. So, if wanted to add any such setting to the main ServerConfig class, we'd just add it there something like this:

```
pubsub:
   endpoint: http://not.the.normal.google.com
// or global
google:
   endpoint: http://not.the.normal.google.com
```

That said, maybe we should punt on all that. There's another way to set this up that just uses an env var, I can just do that instead and remove the APIClientOptions. See: https://pkg.go.dev/cloud.google.com/go/pubsub#hdr-Emulator

Reply:
Done